src/components/DocumentStatus/PromptDefault.tsx

```diff
@@ -30,7 +31,13 @@ export const PromptDefault = ({ prompt, setStatus, showPrompt, requireCause = fa
       }}
       disablePrimary={requireCause && !cause}
     >
-      {requireCause && (
+      {cause && (
+        <PromptCauseField
+          onValueChange={setCause}
+          cause={prompt.status !== 'draft' ? cause : ''}
+        />
+      )}
+      {requireCause && !cause && (
         <div className='flex flex-col gap-2'>
           <PromptCauseField onValueChange={setCause} />
         </div>
```

Comment:
This seems some what duplicated? Enough with something like this:
```suggestion
      {(cause || requireCause) && (
        <div className='flex flex-col gap-2'>
          <PromptCauseField
            onValueChange={setCause}
            cause={prompt.status !== 'draft' ? cause : ''}
          />
        </div>
      )}

```